### PR TITLE
A new readme file with a screenshot

### PR DESCRIPTION
**Pull request by navintc**
A new screenshot added.
Replaced the old readme with a new readme.md file.
Added the screenshot into the readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
-you need gcc4 to compile it. That means, if you are using a hybrid perhaps you need to run the command "segcc gcc4" first.
-You can compile now the program by writing "jam" in the terminal from inside the source-directory.
-
-You need at least r1ac3 rc   to be able to run the application without major problems.
-
-The source code is very dirty. It needs to be cleaned and restructured and some things reimplemented. I did most of the work late at night and under time pressure I decided to implement the solution that was the fastest to implement. I hope this summer i will have some time to clean it
-
-.
\ No newline at end of file
+README
```
Comment (humdingerb) [inline README.md]: Maybe "Puri" is a better heading. Use markdown to format it.

File: README.md [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
+
+![Puri screenshot](Screenshot.png "Puri")
+
+you need gcc4 to compile it. That means, if you are using a hybrid perhaps you need to run the command "segcc gcc4" first.
```
Comment (humdingerb) [inline README.md]: Change to "You need at least gcc4 to compile Puri."
The command is "setarch x86".

File: README.md [hunk 3]
```diff
@@ -1,10 +1,12 @@
...
+You can compile now the program by writing "jam" in the terminal from inside the source-directory.
```
Comment (humdingerb) [inline README.md]: Change to:
"You compile the program by writing "`make`" in Terminal from inside the 'source' directory."

File: README.md [hunk 4]
```diff
@@ -1,10 +1,12 @@
...
+
+You need at least r1ac3 rc   to be able to run the application without major problems.
```
Comment (humdingerb) [inline README.md]: Remove the whole sentence pointing to R1Alpha3. That's many years in the past...

File: README.md [hunk 5]
```diff
@@ -1,10 +1,12 @@
...
+
+The source code is very dirty. It needs to be cleaned and restructured and some things reimplemented. I did most of the work 
+late at night and under time pressure I decided to implement the solution that was the fastest to implement. I hope this 
+summer i will have some time to clean it
```
Comment (humdingerb) [inline README.md]: Apparently, he didn't have time that summer, many years ago... :)
Let's remove that whole paragraph, too.

File: README.md (hunk 1 above)
Comment (humdingerb) [reply README.md]: Before the screenshot, add an intro sentence, like:
> Puri is a chess program to play offline or against somebody over the network. It uses the Stockfish chess engine.

Add more info if you find something interesting.

**Review by humdingerb — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (scottmc) [reply README.md]: yup.  it will live on it the commit history, doesn't need to show up in the readme though. ;)

**Comment by navintc** [comment]
I changed the readme.md file. Check it. Tell me if somethings wrong. Thanks.

**Review by humdingerb — APPROVED** [review]

**Comment by humdingerb** [comment]
Thanks. I made some more minor text changes before merging.

**Pull request merged**